temporal-sdk/src/main/java/io/temporal/activity/ActivityInfo.java

```diff
@@ -57,46 +58,32 @@
    */
   long getScheduledTimestamp();
 
-  /**
-   * Returns the Schedule-To-Close Timeout setting as a Duration.
-   */
+  /** Returns the Schedule-To-Close Timeout setting as a Duration. */
   Duration getScheduleToCloseTimeout();
 
-  /**
-   * Returns the Start-To-Close Timeout setting as a Duration.
-   */
+  /** Returns the Start-To-Close Timeout setting as a Duration. */
   Duration getStartToCloseTimeout();
 
-  /**
-   * Returns the Heartbeat Timeout setting as a Duration.
-   */
+  /** Returns the Heartbeat Timeout setting as a Duration. */
   Duration getHeartbeatTimeout();
 
   Optional<Payloads> getHeartbeatDetails();
 
-  /**
-   * Returns the Workflow Type of the Workflow Execution that executed the Activity.
-   */
+  /** Returns the Workflow Type of the Workflow Execution that executed the Activity. */
```

Comment:
If you revisit these docs, let's use Javadoc @return tag instead of plain "Returns"